RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -789,4 +789,54 @@ boolean auto_burnLeaves()
 		return false;
 	}
 	return false;
+}
+
+boolean auto_haveCCSC()
+{
+	if(auto_is_valid($item[Candy Cane Sword Cane]) && (item_amount($item[Candy Cane Sword Cane]) > 0 || have_equipped($item[Candy Cane Sword Cane])))
```

Comment:
This is fine but you can use `available_amount()` as it works like `item_amount()` but counts anything currently equipped too.

Reply:
I like available_amount. Makes the conditional cleaner